packages/core/src/lib/sign-in-experience/sign-in.ts

```diff
@@ -31,6 +31,13 @@ export const validateSignIn = (
     );
   }
 
+  assertThat(
+    signIn.methods.every(({ password, verificationCode }) => password || verificationCode),
+    new RequestError({
+      code: 'sign_in_experiences.at_least_one_authentication_factor',
```

Comment:
In the current version, maybe we need piece of detailed information about which method is affected by this rule. @fleuraly will take care of this.